script/before_install.sh

```diff
@@ -165,4 +174,4 @@ then
             fi
         fi
     fi
-fi
\ No newline at end of file
+fi
```

Comment:
Consider reverting this unnecessary whitespace change.

Reply:
Oh wait, you added a new line in the end. Well, I guess this is less of an issue then. But still, try to avoid whitespace changes in areas of the code you did not touch.